eo-maven-plugin/src/test/java/org/eolang/maven/ProbeMojoTest.java

```diff
@@ -132,17 +116,17 @@ void findsProbesViaOfflineHash(@TempDir final Path temp) throws IOException {
     @ExtendWith(OnlineCondition.class)
     void findsProbesInOyRemote(@TempDir final Path temp) throws IOException {
         final String tag = "0.28.10";
-        final CommitHash hash = new ChRemote(tag);
         MatcherAssert.assertThat(
-            ProbeMojoTest.firstEntry(
-                new FakeMaven(temp)
-                    .with("tag", tag)
-                    .with("objectionaries", new Objectionaries.Fake(new OyRemote(hash)))
-                    .withProgram(ProbeMojoTest.program())
-                    .execute(new FakeMaven.Probe())
-                    .foreignPath(),
-                "probed"
-            ),
+            new FakeMaven(temp)
```

Comment:
@maxonfjvipon I thought about it too and decided that it would be better to do in the scope of that issue:
https://github.com/objectionary/eo/issues/2330